app/presenters/dashboard_presenter.rb

```diff
@@ -0,0 +1,19 @@
+# typed: strict
+# frozen_string_literal: true
+
+# The page model for the dashboard
+class DashboardPresenter
+  extend T::Sig
+
+  sig { params(drafts: ActiveRecord::Relation, collections: ActiveRecord::Relation).void }
```

Comment:
Can you say more about the distinction between a presenter and a view component, conceptually?

Reply:
A presenter is a model of the data and a view component is a model of the view. This comes from Sandi Metz's rules of thumb:
> Controllers can instantiate only one object. Therefore, views can only know about one instance variable and views should only send messages to that object (@object.collaborator.value is not allowed).

Reply:
So, for instance, we wouldn't see a component instantiated from within a controller. But we would a presenter. OK.